p4include/v1model.p4

```diff
@@ -28,6 +28,7 @@ match_kind {
 }
 
 // Are these correct?
+@metadata @name("standard_metadata")
```

Comment:
I think we're better off making this data driven. I'm trying to get to the point where we have little or no backend-specific stuff baked into the PI serialization code. (Though we've got a ways to go before we're at that point.)